gnomad_mitochondria/utils/exports.py

```diff
@@ -1,5 +1,10 @@
 import hail as hl
 
+from collections import Counter
```

Comment:
odd to be using something very Python-y in Hail . 1) are we even thinking about touching the code now or is this just rearranging existing code 2) would there be a better way to do this that doesn't need to call .collect() ?

Reply:
mostly just rearranging existing code, Counter is only used to count the number of variants with a particular filter and export to a text file rather than another Table, so I think it's fine in this case, although also could be done with an aggregation